application/backend/src/main/resources/apps/typerefinery/components/content/columnticker/dialog/.content.json

```diff
@@ -0,0 +1,66 @@
+{
+    "sling:resourceType": "wcm/dialogs/dialog",
+    "tabs": {
+        "sling:resourceType": "wcm/dialogs/components/tabs",
+        "generalTab": {
+            "sling:resourceType": "wcm/dialogs/components/tab",
+            "label": "General",
+            "title": {
+                "sling:resourceType": "wcm/dialogs/components/textfield",
+                "name": "title",
+                "label": "Title"
+            },
+            "icon": {
+                "sling:resourceType": "wcm/dialogs/components/textfield",
+                "name": "icon",
+                "label": "icon"
+            }
+        },
+        "dataTab": {
+            "sling:resourceType": "wcm/dialogs/components/tab",
+            "label": "Data",
+            "value": {
+                "sling:resourceType": "wcm/dialogs/components/textfield",
+                "name": "value",
+                "label": "Value"
+            },
+            "indicatorType": {
```

Comment:
Please remove unwanted codes from here.

I can see few things which are not used.

Reply:
Done